LucidGloves/include/Quaternion.h

```diff
@@ -15,5 +15,6 @@ vr::HmdVector3_t GetPosition(vr::HmdMatrix34_t matrix);
 vr::HmdQuaternion_t MultiplyQuaternion(const vr::HmdQuaternion_t q, const vr::HmdQuaternion_t r);
 
 vr::HmdQuaternion_t QuaternionFromAngle(const double& xx, const double& yy, const double& zz, const double& a);
+vr::HmdQuaternion_t EulerToQuaternion(const double& xx, const double& yy, const double& zz);
```

Comment:
Can these be the same argument/parameter names as what's provided in the definition?